### DISCUSSION TITLE
ipv6 only

### DISCUSSION THREAD

**Discussion by vulnfind** [postalserver/postal#2425] [Help with using Postal]
Hello Everyone

I have a single ipv4 at which the postal server is hosted. I have assigned an ipv6 to the server too. Now I have a pool of ipv6 /64 which I want to use for sending mails. I want the mail server to use ipv6 only when sending mails but the option in the UI shows the option to add ipv6 but it also binds ipv4 to it which doesn't make sense.

Anyone here can help would be much appreciated. Thanks in advance

**Comment by willpower232** [comment]
Are you trying to use IP Pools to selectively use your available IPv6 address? I don't think there is an IPv6 only mode that I am aware of.
